tests/load/pipeline/test_merge_disposition.py

```diff
@@ -206,6 +206,23 @@ def test_merge_keys_non_existing_columns(destination_name: str) -> None:
         assert "m_a1" not in table_schema  # unbound columns were not created
 
 
+@pytest.mark.parametrize('destination_name', ["duckdb", "snowflake", "bigquery"])
+def test_pipeline_load_parquet(destination_name: str) -> None:
```

Comment:
This only tests merge disposition, right?  
I would add replace test as well. Maybe add `loader_file_format` params to some existing replace tests, e.g. `[(dest, None) for dest in ALL_DESTINATIONS] + [('duckdb', 'parquet'), ('snowflake', 'parquet')]`